### PR TITLE
Python miniapp

### PR DESCRIPTION
**Pull request by apeyser**
Run the miniapp from python as pyminiapp.miniapp(arguments....)

### CODE REVIEW

**Comment by apeyser** [comment]
Primarily, I'd like people's opinions on directory structure & cmake infrastructure. The code itself is trivial.

**Comment by apeyser** [comment]
Now it's been converted from simply applying the command line to creating an object for setting command line-type parameters.

File: python/miniapp/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+set(PYMINIAPP_SOURCES
```
Comment (w-klijn) [inline python/miniapp/CMakeLists.txt]: I would expect the same pattern to develop as the c++ side: multiple miniapps. So the directory structure could mimic this?

Comment (apeyser) [reply python/miniapp/CMakeLists.txt]: Yes, this is what I've done on the feature/python branch which add pyminiapp + pyarbor

**Comment by bcumming** [comment]
Closing this because it has rotted, and work on using pybind11 is now under way.

**Pull request closed without merging**